project2/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Overpass:400,700|Questrial|Righteous&display=swap"
+        rel="stylesheet">
+    <script src="https://kit.fontawesome.com/f7ec3810d9.js" crossorigin="anonymous"></script>
+    <title>MOONVIES</title>
+</head>
+
+<body>
+    <h1>MOONVIES</h1>
+    <nav class="nav">
+        <ul>
+            <li><a href="movienav">MOVIES</a></li>
+            <li><a href="shownav">SHOWS</a></li>
+            <li><a href="reviewnav">REVIEWS</a></li>
+            <li><a href="aboutnav">ABOUT US</a></li>
+            <li><a href="accountnav">ACCOUNT</a></li>
+        </ul>
+    </nav>
+
+
+
+    <section class="cardbox">
+        <img src="images/infin.jpg">
+        <div class="content">
+            <h2>AVENGERS: ENDGAME</h2>
+            <div class="description">
+                <div class="ratingsection">
+                    <p>Joe Russo</p>
+                    <i class="fas fa-circle"></i>
+                    <i class="fas fa-circle"></i>
+                    <i class="fas fa-circle"></i>
+                    <i class="fas fa-circle"></i>
+                    <i class="fas fa-adjust"></i>
+                    <p class="rating">4.5</p>
```

Comment:
For a user experiencing the web via a screenreader, they won't have the context of the circles/stars, so they won't know if this 4.5 rating is out of 5 or 10 or 20 possible units.